bigtable/client/client_options.h

```diff
@@ -53,6 +53,126 @@ class ClientOptions {
     return *this;
   }
 
+  /**
+   * Set Compression Algorithm for channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetCompressionAlgorithm()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  inline void SetCompressionAlgorithm(grpc_compression_algorithm algorithm) {
```

Comment:
With modern compilers, I believe you can skip `inline` because compilers ignore these directives and make their own decisions, and may inline things that are not marked `inline`, and NOT inline things that ARE marked `inline`.

Up to you and @coryan if you want to keep or remove these directives (I recommend dropping them).

See also [Wikipedia page][wikipedia]:

> [...] an inline function is one qualified with the keyword `inline`; [...] it serves as a compiler directive that **suggests (but does not require)** that the compiler substitute the body of the function inline by performing inline expansion, i.e. by inserting the function code at the address of each function call, [..]

(emphasis mine)

[wikipedia]: https://en.wikipedia.org/wiki/Inline_function

Reply:
@coryan thoughts?

Reply:
Please remove the `inline` keywords.  They are not needed in member functions.

Reply:
I think this is the only outstanding review comment at this time, if I'm not mistaken.

Reply:
Done.